hypervisor/arch/x86/Kconfig

```diff
@@ -8,7 +8,9 @@ config SCENARIO
 	  one post-launched KATA VM, one post-launched RT VM for real-time control,
 	  and up to five post-launched Standard VMs;
 	  hybrid: Typical scenario for hybrid usage with 3 VMs: one pre-launched VM,
-	  one pre-launched SOS VM and one post-launched Standard VM;
+	  one pre-launched Service VM and one post-launched Standard VM;
```

Comment:
Do we need to stress the pre-launched safety VM here?
Previously the naming of hybrid scenario is a generic term that it will load any kinds of pre-launched VM whatever safetyVM or RTVM or standardVM, but with hybrid_rt scenario introduced, seems we need to distinguish them.

Reply:
Is there any special attribute to that pre-launched VM makes it a "Safety VM"?

We can certainly add this in the text though regardless of the answer to the question above 👍

Should we also explain in here that the value of that variable could be yet something different is someone creates their own `misc/vm_configs/scenarios/<new scenario>`?